utils_linux.go

```diff
@@ -378,6 +378,9 @@ func (r *runner) checkTerminal(config *specs.Process) error {
 }
 
 func validateProcessSpec(spec *specs.Process) error {
+	if spec == nil {
+		return fmt.Errorf("process property must not be empty")
```

Comment:
nit: since there's no formatting, it's better to use `errrors.New()`.

You can add a patch just before this one, converting the other places like this.

Reply:
Okay, will do. Thanks for the advice.

Reply:
Hi, I'm a bit unsure what is meant by a patch. Does this relate to git's patch files? A separate issue? A separate commit? Or just another change I apply to the current commit?
Thanks!

Reply:
I mean two separate commits, yes.